src/prefect/projects/steps/pull.py

```diff
@@ -70,4 +70,5 @@ def git_clone_project(
         ) from exc_chain
 
     directory = "/".join(repository.strip().split("/")[-1:]).replace(".git", "")
+    print(f"Cloned repo {repository_url} to directory {directory}")
```

Comment:
Should we write a log message instead?

Reply:
Do you mean something like 
```python
projects_logger = get_logger("projects")
projects_logger.info(f"...")
```
here?

Reply:
@madkinsz I agree something like this would look nice:

```bash
❯ prefect worker start -p my-worker --type process
Worker 'ProcessWorker 541fe972-f03d-4a70-bb8d-e14184ac55c4' started!
13:30:04.757 | INFO    | prefect.worker.process.processworker 541fe972-f03d-4a70-bb8d-e14184ac55c4 - Submitting flow run '00883294-e0a5-492c-8b4e-c75d74c6a430'
13:30:05.686 | INFO    | prefect.worker.process.processworker 541fe972-f03d-4a70-bb8d-e14184ac55c4 - Opening process...
13:30:06.098 | INFO    | prefect.worker.process.processworker 541fe972-f03d-4a70-bb8d-e14184ac55c4 - Completed submission of flow run '00883294-e0a5-492c-8b4e-c75d74c6a430'
Cloning into 'prefect'...
Updating files: 100% (1293/1293), done.
13:30:33.609 | INFO    | prefect.projects - Cloned repository 'https://github.com/PrefectHQ/prefect.git' into 'prefect'
13:30:33.652 | ERROR   | Flow run 'zircon-fox' - Flow could not be retrieved from deployment.
```

Reply:
@gabcoyne You can achieve this by adding
```python
projects_logger = get_logger("projects")
```
at line 10

and modifying your current code at line 73 with
```python
projects_logger.info(f"Cloned repository {repository!r} into {directory!r}")
```

Reply:
Agree